src/tools/base/BaseBrushTool.js

```diff
@@ -3,10 +3,8 @@ import EVENTS from './../../events.js';
 import BaseTool from './BaseTool.js';
 import isToolActive from './../../store/isToolActive.js';
 import store from './../../store/index.js';
-import { getToolState } from '../../stateManagement/toolState.js';
-import { globalImageIdSpecificToolStateManager } from '../../stateManagement/imageIdSpecificStateManager.js';
 
-const { state, setters } = store.modules.brush;
+const { configuration, getters, setters } = store.modules.segmentation;
```

Comment:
- [ ] Migration Note

These are technically breaking changes, as we use hardcoded namespaces?

Reply:
Yeah, there is a lot of `brush` => `segmentation` throughout, which is all breaking. We originally only had brush tools, but the module is useful for all segmentation tools.